.circleci/config.yml

```
@@ -0,0 +1,79 @@
+version: 2.1
+
+gpu: &gpu
+  environment:
+    CUDA_VERSION: "10.1"
+  machine:
+    image: ubuntu-1604-cuda-10.1:201909-23 
+  resource_class: gpu.large
+
+
+jobs:
+  test_nvidia_smi:
+    <<: *gpu
+    steps:
+      - checkout
+      - run: nvidia-smi
+  flexflow_build_test:
+    <<: *gpu
+    steps:
+      - checkout
+      - run:
+          name: "Set Python Version"
+          command: |
+            wget -c https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+            mv Miniconda3-latest-Linux-x86_64.sh ~/Miniconda3-latest-Linux-x86_64.sh
+            chmod +x ~/Miniconda3-latest-Linux-x86_64.sh
+            sudo ~/Miniconda3-latest-Linux-x86_64.sh -b -p /opt/conda
+            rm ~/Miniconda3-latest-Linux-x86_64.sh
+            sudo /opt/conda/bin/conda upgrade --all
+            sudo /opt/conda/bin/conda install -y conda-build conda-verify
+            sudo /opt/conda/bin/conda clean -ya
+            sudo /opt/conda/bin/conda install -y cmake make pillow
+            sudo /opt/conda/bin/conda install -y -c conda-forge protobuf=3.9 numpy keras-preprocessing
+      - run: 
+          name: Install dependencies
+          command: |
+            cd nccl
+            git submodule update --init --recursive
+            make -j src.build NVCC_GENCODE="-gencode=arch=compute_60,code=sm_60"
+            cd ..
+      - run: 
+          name: Build FlexFlow
+          command: |
+            export FF_HOME=$PWD
```

I don't think export is required.
```suggestion
            FF_HOME=$PWD
```

Ditto for the others.